prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/git/GitUtils.java

```diff
@@ -0,0 +1,39 @@
+package com.redhat.parodos.tasks.git;
+
+import com.redhat.parodos.workflows.work.WorkContext;
+import lombok.Getter;
+
+import static com.redhat.parodos.workflow.context.WorkContextDelegate.getOptionalValueFromRequestParams;
+
+public class GitUtils {
```

Comment:
Utils classes should comply with coding convention, see:
https://github.com/spring-projects/spring-framework/wiki/Code-Style#utility-classes

> 
> A class that is only a collection of static utility methods must be named with a Utils suffix, must have a private default constructor, and must be abstract. Making the class abstract and providing a private default constructor prevent anyone from instantiating it. For example:
> 
> public abstract MyUtils {
> 
>     private MyUtils() {
>         /* prevent instantiation */
>     }
> 
>     // static utility methods
> }